pkg/minikube/translate/translations/zh-CN.json

```diff
@@ -0,0 +1,9 @@
+{
+    "minikube %s on %s (%s)": "",
+    "Creating %s VM (CPUs=%d, Memory=%dMB, Disk=%dMB) ...": "",
+    "Configuring environment for Kubernetes %s on %s %s": "",
+    "Pulling images ...": "",
+    "Launching Kubernetes ...": "",
+    "Verifying:": "",
+    "Done! kubectl is now configured to use %q": ""
+}
```

Comment:
If you would like to add initial values for zh-CN:

https://beta.etherpad.org/p/minikube__kubecon_shanghai

Otherwise it can be in a follow-up PR.

Reply:
Done.